utilities/segmentation_utils/flowreader.py

```diff
@@ -297,6 +296,9 @@ def __init__(
         self.preprocessing_seed = preprocessing_seed
         self.read_weights = read_weights
         self.weights_path = weights_path
+        self.preprocessing_queue_image = preprocessing_queue_image
+        self.preprocessing_queue_mask = preprocessing_queue_mask
+        self.shuffle_counter = shuffle_counter
```

Comment:
Also as a last comment if the tests have passed this way, we need to invest time into adding more tests and improve our existing ones